### PR TITLE
Add pages for notifications settings

### PR DESCRIPTION
**Pull request by kunstefix**
Add pages for notifications settings

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: extensions/apps/settings/src/components/pages/notifications-browser-option.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  const {
+    data: { authenticatedDID, isAuthenticating },
+  } = useAkashaStore();
+  const isLoggedIn = !!authenticatedDID;
+
+  const { getCorePlugins } = useRootComponentProps();
+  const routingPlugin = useRef(getCorePlugins().routing);
+
+  if (!isLoggedIn && !isAuthenticating) {
```
Comment (didd) [inline extensions/apps/settings/src/components/pages/notifications-browser-option.tsx:31]: If user isn't authenticated then we need to show them a card with info and a connect button. You can see this card used throughout the app, example: https://next.akasha-world-framework.pages.dev/@akashaorg/app-extensions/my-extensions.

File: extensions/apps/settings/src/components/pages/notifications-option.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  const { getCorePlugins } = useRootComponentProps();
+  const routingPlugin = useRef(getCorePlugins().routing);
+
+  if (!isLoggedIn && !isAuthenticating) {
+    // if not logged in, redirect to homepage
+    routingPlugin.current?.navigateTo?.({
+      appName: '@akashaorg/app-antenna',
+      getNavigationUrl: () => '/',
+    });
+  }
```
Comment (didd) [inline extensions/apps/settings/src/components/pages/notifications-option.tsx]: Same comment as the above one.

File: extensions/apps/settings/src/components/pages/notifications-preferences-option.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  const { getCorePlugins } = useRootComponentProps();
+  const routingPlugin = useRef(getCorePlugins().routing);
+
+  if (!isLoggedIn && !isAuthenticating) {
+    // if not logged in, redirect to homepage
+    routingPlugin.current?.navigateTo?.({
+      appName: '@akashaorg/app-antenna',
+      getNavigationUrl: () => '/',
+    });
+  }
```
Comment (didd) [inline extensions/apps/settings/src/components/pages/notifications-preferences-option.tsx]: Same comment as the above one.

File: extensions/apps/settings/src/components/pages/notifications-option.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+          const children = (
+            <>
+              <Text>{`${t('{{itemLabel}}', { itemLabel: item.label as string })}`}</Text>
+              {!item.isSubheading && <Icon icon={<ChevronRightIcon />} accentColor={true} />}
+            </>
+          );
+
+          return (
+            <React.Fragment key={`${idx}${item.label}`}>
```
Comment (didd) [inline extensions/apps/settings/src/components/pages/notifications-option.tsx]: If the data doesn't come with id then you can use `crypto.randomUUID()`

Comment (kunstefix) [reply extensions/apps/settings/src/components/pages/notifications-option.tsx]: Updated. Labels are unique in this case, so I just used them as key.

File: extensions/apps/settings/src/components/pages/notifications-browser-option.tsx (hunk 1 above)
Comment (kunstefix) [reply extensions/apps/settings/src/components/pages/notifications-browser-option.tsx:31]: Updated

File: extensions/apps/settings/src/components/pages/notifications-option.tsx (hunk 2 above)
Comment (kunstefix) [reply extensions/apps/settings/src/components/pages/notifications-option.tsx]: Updated

**Review by didd — APPROVED** [review]

**Review by quininez — APPROVED** [review]

**Pull request merged**